pkg/commands/plan/plan.go

```diff
@@ -313,14 +332,19 @@ func (c *PlanCommand) updateResultCommentForActions(ctx context.Context, startCo
 }
 
 func (c *PlanCommand) getMessageBody(result *RunResult, resultErr error) string {
-	msgBody := fmt.Sprintf("%s 🟦 No changes for dir: `%s` %s", CommentPrefix, c.childPath, c.gitHubLogURL)
+	destroyText := ""
+	if c.flagDestroy {
+		destroyText = DestroyCommentText
+	}
+
+	msgBody := fmt.Sprintf("%s 🟦 No changes for dir: `%s` %s%s", CommentPrefix, c.childPath, c.gitHubLogURL, destroyText)
 
 	if result.hasChanges || resultErr != nil {
 		var comment strings.Builder
 		if resultErr != nil {
-			fmt.Fprintf(&comment, "%s 🟥 Failed for dir: `%s` %s\n\n<details>\n<summary>Error</summary>\n\n```\n\n%s\n```\n</details>", CommentPrefix, c.childPath, c.gitHubLogURL, resultErr)
+			fmt.Fprintf(&comment, "%s 🟥 Failed for dir: `%s` %s%s\n\n<details>\n<summary>Error</summary>\n\n```\n\n%s\n```\n</details>", CommentPrefix, c.childPath, c.gitHubLogURL, destroyText, resultErr)
```

Comment:
Could this be a helper function like `failureMessage(prefix, dir, childPath, url, destroyText string, err error)`? No strong feelings from me, but I see this duplicated around this codebase.

Reply:
100%, but this is tech debt, I think we should implement a Reporter interface and have different ways of writing output. Ill create an issue.

Reply:
Created https://github.com/abcxyz/guardian/issues/355